src/pages/Rockets.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getrocket } from '../redux/rocketsSlice';
+import Rocket from '../components/SingleRocket';
+
+const Rockets = () => {
+  const RocketList = useSelector((state) => state.rocket);
```

Comment:
- [ ] I think the data is being returned by the API but then it's possible there is an error with the way you are accessing the data from the API. Kindly check carefully.

![image](https://user-images.githubusercontent.com/63473584/225765096-33879e95-e2e6-4390-a14c-354ebb6323c4.png)

Reply:
thank you @vickymarz , we fixed the problem.

Reply:
- Well resolved team 👍🏽